test/connect.js

```diff
@@ -118,4 +118,15 @@ describe('Basic Connectivity', function() {
     }, 100 * 2);
   });
 
+
+  it('should add a new cluster server', function(done){
+    var natsServers = [uri]
+    var nc = NATS.connect({servers: natsServers});
+    nc.on('connect', function(client) {
+      client.addServer('nats://localhost:22223');
+      client.servers.length.should.equal(2);
```

Comment:
Ok I see that random ordering happens at the constructor level https://github.com/nats-io/node-nats/blob/master/lib/nats.js#L241 i.e. not upon connect / reconnect so I can run `shuffle` again based on `opts.noRandomize` unless `noRandomize`is passed as a second argument in which case it overrides the default behaviour. Makes sense?